### PR TITLE
Move most dependencies from install_core_deps to chaiNNer_standard Dependency definition

### PR DESCRIPTION
**Pull request by joeyballentine**
these feel like they make more sense here.

### CODE REVIEW

**Comment by joeyballentine** [comment]
this apparently doesn't work? it works fine in npm start but with npm run dev I get this... 
image

File: backend/src/server.py [hunk 1]
```diff
@@ -493,16 +505,15 @@ async def setup(sanic_app: Sanic):
     await setup_queue.put_and_wait(
         {
             "event": "backend-status",
```
Comment (RunDevelopment) [inline backend/src/server.py:507]: Could we use `update_progress` for these "backend-status" events too?

File: backend/src/server.py [hunk 2]
```diff
@@ -419,7 +419,8 @@ def install_deps(dependencies: List[api.Dependency]):
     for package in api.registry.packages.values():
         logger.info(f"Checking dependencies for {package.name}...")
         if package.name == "chaiNNer_standard":
-            continue
+            await update_progress_cb("Installing core dependencies...", 0.65)
+            install_deps(package.dependencies)
```
Comment (RunDevelopment) [inline backend/src/server.py:423]: Why not push them into `to_install`?

File: backend/src/packages/chaiNNer_standard/__init__.py [hunk 3]
```diff
@@ -1,14 +1,30 @@
...
+from api import KB, MB, Dependency, add_package
+from system import is_arm_mac, is_windows
 
 package = add_package(
     __file__,
     name="chaiNNer_standard",
     description="The standard set of nodes for chaiNNer.",
-    dependencies=[],
+    dependencies=[
+        Dependency("OpenCV", "opencv-python", "4.7.0.68", 30 * MB, import_name="cv2"),
```
Comment (RunDevelopment) [inline backend/src/packages/chaiNNer_standard/__init__.py:11]: Could you please use kw args here like we do in other packages?

File: backend/src/dependencies/install_core_deps.py [hunk 4]
```diff
@@ -49,23 +11,10 @@
...
-    {
-        "package_name": "opencv-python",
-        "version": "4.7.0.68",
-    },
-    {
-        "package_name": "Pillow",
-        "version": "9.2.0",
-    },
     {
         "package_name": "chainner_ext",
```
Comment (RunDevelopment) [inline backend/src/dependencies/install_core_deps.py:15]: I think you can move this over too.

**Comment by joeyballentine** [comment]
Assuming this is good now, going to merge so I can open my next PR

**Pull request merged**